packages/enso-browser-gateway/src/voice-forwarder.mjs

```diff
@@ -0,0 +1,64 @@
+export function resolveFrameDurationMs(channel) {
+  if (!channel?.protocol) return 20;
+  const match = /frame(?:Duration)?(?:Ms)?=(\d+)/i.exec(channel.protocol);
+  if (!match) return 20;
+  const value = Number.parseInt(match[1], 10);
+  return Number.isFinite(value) && value > 0 ? value : 20;
+}
```

Comment:
`resolveFrameDurationMs`: different browsers omit `channel.protocol`. Consider a clamp so absurd values (e.g., `<5` or `>200`) fall back to 20ms:

```js
const value = Number.parseInt(match[1], 10);
if (!Number.isFinite(value) || value < 5 || value > 200) return 20;
return value;
```
Add a README line explaining the protocol contract and fallback.